frontend-shared/styles/mixins/focus.scss

```diff
@@ -1,3 +1,26 @@
+@use '../variables' as var;
+
+/**
+ * Add stylized focus to interactive elements such <input> or <textarea>
+ *
+ * @param {boolean} $inset - Does the outline render inset or not
+ */
+@mixin outline($inset: false) {
+  &:focus {
+    outline: none;
+    @if $inset {
```

Comment:
Might check out the SASS `if` function for alternate (potentially more concise) syntax here, but it's not a big deal: https://sass-lang.com/documentation/modules#if